packages/webapp/socket_file.js

```
@@ -51,11 +55,25 @@ export const removeExistingSocketFile = (socketPath) => {
 // process is killed via signal 9 - SIGKILL.
 export const registerSocketFileCleanup =
   (socketPath, eventEmitter = process) => {
+    // Remove previously registered handlers for this emitter to prevent
```

ESLint

[error] Delete `··`

(prettier/prettier)